nessie/hooks_test.go

```diff
@@ -145,6 +186,26 @@ func TestHooksSuccess(t *testing.T) {
 	require.Equal(t, "main", run.Branch)
 }
 
+func parseAndUpload(t *testing.T, ctx context.Context, repo, branch, yaml, templateName, actionPath string) {
```

Comment:
call `t.Helper()` at start so we will have the error report the caller as the test that failed